tests/gold.c

```diff
@@ -0,0 +1,15 @@
+// Test gold support
+// REQUIRES: clang
+// RUN: %clang -fuse-ld=gold %s -o %t
+// RUN: %t
+
+#include <stdio.h>
+int main() {
+if (1==1) {
```

Comment:
I wonder what you're trying to test with that trivial comparison?

Reply:
I just took the code coverage test. The code is unrelevant here :)